libpromises/eval_context.c

```diff
@@ -1058,6 +1085,11 @@ ENTERPRISE_VOID_FUNC_1ARG_DEFINE_STUB(void, EvalContextSetupMissionPortalLogHook
 {
 }
 
+void DestroyNothing(void *data)
```

Comment:
Otherwise, it will complain about unused arguments when compiling with `NDEBUG`
```suggestion
void DestroyNothing(NDEBUG_UNUSED void *data)
```